enumerable.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './my_enumerable'
+class MYList
```

Comment:
- [ ] Hey @fuadmoin you have done well implementing some of the requirements for this task but I would suggest that you change the class name to adhere to the ruby naming rules instead of `MYList` it should be `MyList `